### ISSUE TITLE
Certificate for OpenOTP Two Factor Authentication

### ISSUE THREAD

**Issue by rcdevs** [owncloud/appstore-issues#172]
appid: openotp_auth
repo: https://github.com/rcdevs/openotp_authentication_owncloud

-----BEGIN CERTIFICATE REQUEST-----
MIIEXDCCAkQCAQAwFzEVMBMGA1UEAwwMb3Blbm90cF9hdXRoMIICIjANBgkqhkiG
9w0BAQEFAAOCAg8AMIICCgKCAgEAxH4bPhD0iMgny8O421BTG1lSNzZcVrRnRdH/
LYsn+0dVUr9fpBfKmRkYFAgyFwnXcjNMevGREV+e5DkhADD1wdCuu+kDckwFqE2H
Chjt4+43IUV+BFVkTqyB4nuRG9SB9GivEsTw1NuSg+UxfmNWuW8O5oB0tSzWomTk
bVsT47QyRf47N/RqQImsGqs449+BgNt1aUdwccqu1qlpR4/1lpQ+kpPvXcaJjWA/
FdPJidS6sHyxPc3d2z65ZmVDnEprdd1RMfhxRSLqJpf3rHEKEtS+Q649ppyCmqOu
t0IKr7FPyz2+hj2cYY5fJbGHFJh5P01cAFuws5KnRKneQEMALwOPBVT9h29FB8Pt
7Z+C868Q9QrA4NlOXK7g2LJufyLvFZqc7Zq3ewxsDgX+YLyVkZaUGpOo2DnA+1ES
G/6vabG+dIP1RsbyC8d0Ip0vIq3RCFfYVkkRmMR3WqWAqHCoSt1kxNMLyi101o4T
2HpdNHCdOM8LYVX6E9gMwYw8H5dHwx2qsoH7LCiBhnZKDjwdO7MkZA4xFnSM8+nf
LX0gM1xV0KN+w0+9LUcfNP2Pxw9cLSYs+Iyo7TcXkI+x2L2/GoTsj+xrpdlYr/ax
ZI3fAfYWWafmlyMpLbzOW6rlpbvbQH3RdKsDGJONRwVc/kIlHW57GzMNEbaD3PGH
9PtpXCsCAwEAAaAAMA0GCSqGSIb3DQEBCwUAA4ICAQBsGLPgQzowxY/aDL7oXe2F
AWSKyLzBWTBXD/8Yssj6ToYmoflL7cA4ywFMzHQ7dy1blB8qBcjpZiy9tNY0/7pA
n5E9IaiOLlB9n+3b3XBah7l+wN/RJvAhM+Q8pJqd1K/Fzx3JokvsseDFZEaAiflQ
jCrCsMTbRmr9x+yW5qrm2GM/M5E/BLIOQi52tA3by8v0zrnCg6+NHC3UW4doaHCy
/UGWD+Gze42r5U8Rf0/H8En0TVi4E6g5WOIHAgtUzksIVMRUA412iayc19bRzFFk
aRUvSwsG/Ylt3cpYT8KX+IkfoWr6M8+s60jnlAh6YI5xK4Pr83VYKb6237Ne1hf3
VJFfPP8LIVi2f2wvT7drnYwQuOqRH2ISVQ52HpvNXWuja7r57TUSnYkzl1CWAKwE
hTBFoxBSbY/rRU7BAEr986LVRoOL2jtZSp718tDcWxtm7FzLenuWBTyefxZn1FI8
0ZEuDEJ5UwIAFjBvd7a9xWqMrlSSy5YAgG2Z3LG64zH/ZMXxKIudcqaQBqfHgp5b
AwH4VE/1Orw9ppyv/t36+DVdK9o24QirOEehmg+ZA/wpGOlhu6byMNH8iokFyD+z
NsN6O42MbrGSEXHyTQjb+ZxFaa0IOv4aOgDavPTwowUU6h1xYOZlZlsg4NFbaIYg
Vk9Jy4CQZQCX8evHuxD0kg==
-----END CERTIFICATE REQUEST-----

## Certificate

```
-----BEGIN CERTIFICATE-----
MIIE+TCCAuECAhCeMA0GCSqGSIb3DQEBCwUAMG0xCzAJBgNVBAYTAlVTMQ8wDQYD
VQQIDAZCb3N0b24xFjAUBgNVBAoMDW93bkNsb3VkIEluYy4xNTAzBgNVBAMMLG93
bkNsb3VkIENvZGUgU2lnbmluZyBJbnRlcm1lZGlhdGUgQXV0aG9yaXR5MB4XDTIx
MTIyODExMDkyM1oXDTMxMTIyNjExMDkyM1owFzEVMBMGA1UEAwwMb3Blbm90cF9h
dXRoMIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAxH4bPhD0iMgny8O4
21BTG1lSNzZcVrRnRdH/LYsn+0dVUr9fpBfKmRkYFAgyFwnXcjNMevGREV+e5Dkh
ADD1wdCuu+kDckwFqE2HChjt4+43IUV+BFVkTqyB4nuRG9SB9GivEsTw1NuSg+Ux
fmNWuW8O5oB0tSzWomTkbVsT47QyRf47N/RqQImsGqs449+BgNt1aUdwccqu1qlp
R4/1lpQ+kpPvXcaJjWA/FdPJidS6sHyxPc3d2z65ZmVDnEprdd1RMfhxRSLqJpf3
rHEKEtS+Q649ppyCmqOut0IKr7FPyz2+hj2cYY5fJbGHFJh5P01cAFuws5KnRKne
QEMALwOPBVT9h29FB8Pt7Z+C868Q9QrA4NlOXK7g2LJufyLvFZqc7Zq3ewxsDgX+
YLyVkZaUGpOo2DnA+1ESG/6vabG+dIP1RsbyC8d0Ip0vIq3RCFfYVkkRmMR3WqWA
qHCoSt1kxNMLyi101o4T2HpdNHCdOM8LYVX6E9gMwYw8H5dHwx2qsoH7LCiBhnZK
DjwdO7MkZA4xFnSM8+nfLX0gM1xV0KN+w0+9LUcfNP2Pxw9cLSYs+Iyo7TcXkI+x
2L2/GoTsj+xrpdlYr/axZI3fAfYWWafmlyMpLbzOW6rlpbvbQH3RdKsDGJONRwVc
/kIlHW57GzMNEbaD3PGH9PtpXCsCAwEAATANBgkqhkiG9w0BAQsFAAOCAgEAgAue
jDqBs7is/u6gqCa9IrZEvgaR/583fbgpNpTkOWBIM6jWXjeypfLdF1T3FmOZygSO
oeYhSaOKjjUX8CTXUOEccXUi1sPg/kPLjHEWjd1KFPJo7x8oZXoqUylqwWkQ1h4G
cpOBLkDUiVC5rt4hxqLRK/NFkktG0AKeWJ0qub45BTlekS9LosVCkQ569vk5yDh/
8+2vOUQKApDBqDW3ixmAtWIGA2GVSJao4gnWHJ9JEzIlAbaFuDlU0SQ1lnvY+O67
071L/HrQ/zUFFlGZNMuc4y7wJKtFjORxnILF3dN0m4ri14Zq4wv66SDB+/S1KHie
ukxrMq/kPiFcyAff+F52Q78VeVXbU35H+/Tgsu3tCjirdeFQqZnrqOHTKv/Nzk+o
JKIfgHcH5Ug1VY8LlVzIKl5Ac9xXT8ZnVv1qAHuhrGIWY92QGGriyDIbmh1S4h0o
60W9+YEnX/53Hm+uipStxgVUjhOCniLvIhCVrC2/5IKXSpElexC1ZEsRZxSnjSfW
dLZe+72xny126XUCKf7OwOThncdqEJ9/ytk2Dj4OBP7A/Uj/ZodKGyUTtoDGAaWD
pK9GKgXn4MVvzw0X52QyevFJGvz+1dAtrk0V+WeKV6a1VdCUQIGg7kRjnnQS/e1T
opdUztqlJ/NzRN7DFu9MsRCxR1+x4jpYNFr3LE4=
-----END CERTIFICATE-----
```

**Comment by rcdevs** [comment]
Hello. Still waiting for the certificate. Did you have a notification regarding our request ?

**Comment by micbar** [comment]
This was already in our system. So I assume you already have the key. It was submitted 4 years ago.

Here is the cert

```
-----BEGIN CERTIFICATE-----
MIIFBDCCAuwCAhBSMA0GCSqGSIb3DQEBCwUAMG0xCzAJBgNVBAYTAlVTMQ8wDQYD
VQQIDAZCb3N0b24xFjAUBgNVBAoMDW93bkNsb3VkIEluYy4xNTAzBgNVBAMMLG93
bkNsb3VkIENvZGUgU2lnbmluZyBJbnRlcm1lZGlhdGUgQXV0aG9yaXR5MB4XDTE4
MDQwOTE4NDMzMloXDTI4MDQwNjE4NDMzMlowIjEgMB4GA1UEAwwXdHdvZmFjdG9y
X3JjZGV2c29wZW5vdHAwggIiMA0GCSqGSIb3DQEBAQUAA4ICDwAwggIKAoICAQDM
FjjwHUQLjhKDK9cTHG7rvqMefQWXJdCq90CQ9FxdLsOnKgvtrlgw8nCS0j+YeVVy
Zq08goC8MsluBkbixuGrLj+LEt9BNyvQffXTJapIk8K/USSn/tCQqh1YRBZT1t8M
RxkqOxrzBBlVjUginehLrHx2205qVG5fAORrXzRZVA3K56lLZoSBXjHENwrqoc1h
GzXit9kbuqEkSc+tksqnb5YIUq2jhfz4kuP9BQy8UNOzbh1cwqMRURDyKWRgaFdT
7jqUKr/8X3mZF7n+Ch0EktAxkP8Irw07/wU8fp2pO7Oq9LFHtqYg/XMwajGzk8xh
qoWF/Xj7A3GqsakgEe8q4G1YFLKzNZ+V0uRS/rr3dtAUsfD7VfqidIwooCxAII95
bygMeST2vSzruksVJWxamlb98vUB2PaUAe1DC1eOZ5EzVjJxEzz2FiqV2gKuUarL
bB61CX3O4oSv2OKoEx3T3vg4B4i7Fq1f1eUXxMLL9MWHf0O+flzTkcb6znbgXTcY
PdTBMYA9X0ehYjyttUup8yBawEf0RwTU6kaB2lvynlSSLbjeIGa5rJBJObKjQz3i
1F/wqcV88o6gAhts6dHqNWPzTuJJY6cNZOrjufuRUsnsT8zldWJnrhbuHVUyPBn+
qB933PQxE4K6gqA1njoWRPflhMBZC/yJwAwvdbtmqQIDAQABMA0GCSqGSIb3DQEB
CwUAA4ICAQAeYmbL2JP4ritpvvwS6UVqk9I7fqhbe8IeQeWN0pp6Y415UjBO4QX0
okkMFthcPAtgHQtYxtI5o1hnX0gJMZGSEJAIwOePZwvZVG4D6ZOGlZCL8fzwrZ11
4BFRybTqJpvAC43yB8NnkIdsY6f2ClDPpeOCZSnvWM80/DK99Qdz/9TDX4EK/IKR
SOS1RalB63EtsLKX7lRk5E8E09WfkHIct4B8xCWoczyoWUlUoHDq++/LCL1wobKO
UkLlexipQn4HnAiscgq+NQIAkT0wxH/Y/rSZfGix6ZdNHBbAP6OKzccQVsDPqXEI
0f6ZN3leyBoPguFa6nwnQOLFEZUkyLd3VELKogYFW60yIqGMRRnbLhPzkBrdDFWp
Pq8nw0FxpcsjiAkEnbRKlVVioFBZNK/BZnF2bC7gxHP/Ykdz/hmuOw+rXG9WV9tJ
iZcKx+ut/UdjLB//iW6a0UeBS9ja3XWddtDyRNKL3zIYEzLZKoeATHSsqqG2zSk5
P5oPEKPXJN2DwM8tUOXX7AmnGG4C95BONOaPdWTGcOw+/wx/gbG4SV/OhRXUa63o
IaoFzdU+nSX7FJWzMGtZPI6m/qVgymOM/ai3ihZp8jJLmFb3Lw+pWB/H1mdWCPwE
NoaW0KEdZ2vTTHtJmnDpd+viVr8gULFqmI3Zu4mDIfn9Yvk5kIZlXw==
-----END CERTIFICATE-----
```

**Comment by micbar** [comment]
Took me a while to figure this out. Sorry for the delay.

**Comment by rcdevs** [comment]
Hi. Thanks but we do not have it because the employee in charge of this application is not part of the company anymore and so we do not have the private key. We also decided to change the appid (sorry, I had to update the first message and did it today only).

So we would like a certificate for this new app id and we will publish the new version of the app on the marketplace with this new app id. It would then be needed to remove (it can be done already now) the old version on the store, on the old app id (twofactor_rcdevsopenotp). We also do not have the access to the marketplace as the employee did not use the intended corporate account to register the app on the store...
